pkg/apis/kops/validation/validation.go

```diff
@@ -829,6 +829,13 @@ func validateKubeAPIServer(v *kops.KubeAPIServerConfig, c *kops.Cluster, fldPath
 		}
 	}
 
+	if v.ServiceAccountIssuer != nil && len(v.ServiceAccountIssuers) > 0 {
```

Comment:
not sure is this correct way of handling this? I would not like to deprecate the api field either.